Light/include/light/rendering/renderer.hpp

```diff
@@ -16,23 +16,28 @@ namespace Light
 		static void init();
 		static void onWindowResize(uint32_t width, uint32_t height);
 
-		static void beginScene(Camera& camera, glm::mat4 camera_view);
+		static void beginScene(Camera &camera, glm::mat4 camera_view, glm::vec3 camPos);
```

Comment:
Like I mentioned before, we usually take `&` to be a part of the type. Same goes for the following changes in the file (lines 23 - 28).